python/cuml/dask/datasets/classification.py

```diff
@@ -253,7 +253,7 @@ def make_classification(n_samples=100, n_features=20, n_informative=2,
                for idx, f in enumerate(parts)]
 
     X_dela = _create_delayed(X_parts, dtype, worker_rows, n_features)
-    y_dela = _create_delayed(y_parts, dtype, worker_rows)
+    y_dela = _create_delayed(y_parts, np.int, worker_rows)
```

Comment:
shouldn't this be more specifically either `int32` or `int64`?

Reply:
`np.int` resolves to `int64`, I believe.. should I make it explicit?

Reply:
yeah, explicit would be better IMO, look at this:

```python
>>> import numpy as np
>>> np.int
<class 'int'>
>>> np.int32
<class 'numpy.int32'>
>>> np.int64
<class 'numpy.int64'>
>>> np.int == np.int64
False
>>> np.int == np.int32
False
```